src/plugins/npcs/al-kharid/karim-plugin.ts

```diff
@@ -0,0 +1,54 @@
+import {npcAction} from "@server/world/actor/player/action/npc-action";
+import {dialogueAction, DialogueEmote} from "@server/world/actor/player/action/dialogue-action";
+import {ActionType, RunePlugin} from "@server/plugins/plugin";
+import {npcIds} from "@server/world/config/npc-ids";
+import {itemIds} from "@server/world/config/item-ids";
+import {widgets} from "@server/world/config/widget";
```

Comment:
1. Please change imports to have spaces between the curly brackets (ie `import { widgets }` instead of `import {widgets}`
2. Please use single quotes instead of double. :)